### DISCUSSION TITLE
cli: tsinfer  infer --log-section

### DISCUSSION THREAD

**Discussion by chlangley** [tskit-dev/tsinfer#623] [Q&A] [answered]
I would appreciate it someone could direct me to a useful example or more detailed info on how to use the "log-section" option.  
Cheers, Chuck

**Comment by hyanwong** [comment, marked as answer]
I think you use it like this:

```
python3 -m tsinfer infer -v --log-section tsinfer.formats myfile.samples
```

where you can use `tsinfer.inference`, `tsinfer.formats`, or `tsinfer.threads` after the `--log-section` argument. The '-v` says to output login information when running, (`-vv` will be even more verbose), then the `--log-section` argument restricts the output to information from different parts of the process. It's really intended for debugging, less so for general users, I suspect.
